suite-native/module-passphrase/src/components/EnterPassphraseOnTrezorButton.tsx

```diff
@@ -40,8 +50,14 @@ export const EnterPassphraseOnTrezorButton = () => {
         dispatch(onPassphraseSubmit({ value: '', passphraseOnDevice: true }));
     };
 
+    if (!deviceModel) return null;
+
     return (
-        <Button onPress={handleSubmitOnDevice} colorScheme="tertiaryElevation0">
+        <Button
+            onPress={handleSubmitOnDevice}
+            colorScheme="tertiaryElevation0"
+            viewLeft={<Icon name={trezorIconNameMap[deviceModel]} />}
```

Comment:
Could not we use `DeviceModelIcon` here?

Reply:
https://github.com/trezor/trezor-suite/pull/12502/commits/34d3ee7c8f5c3b628e3499b6a28559fd97ba1480